public/src/main/java/com/revenuecat/purchases/logging/VerboseLogHandler.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases.logging
+
+/**
+ * Interface that allows handling logs manually.
+ * See also [Purchases.logHandler]
+ */
+interface VerboseLogHandler {
+    fun log(
+        level: LogLevel,
+        tag: String,
+        message: String,
+        throwable: Throwable?,
+        elements: Array<StackTraceElement>?
+    )
+}
```

Comment:
Please note this interface looks different than the `LogHandler` one. I thought this was better so we could change the LogLevel in the future without breaking implementations